### DISCUSSION TITLE
Plural List Title

### DISCUSSION THREAD

**Discussion by derins** [adityatelange/hugo-PaperMod#338] [Q&A] [answered]
List title is always plural. How would I change it to singular as when my title is literature - it turns into literatures on the list page

**Comment by jeffmhastings** [comment, marked as answer]
I found two ways of doing this.

1. Set [pluralizeListTitles](https://gohugo.io/getting-started/configuration/#pluralizelisttitles) to false in the Hugo configuration

  

   **config.yaml**

   ```yaml
   pluralizeListTitles: false
   ```

   This will apply to all section list pages.

2. Use a section [index page](https://gohugo.io/content-management/organization/#index-pages-_indexmd)

   Create a file `_index.md` inside the section folder and set `title` in the front matter

   **content/literature/_index.md**

   ```md
   ---
   title: Literature
   ---
   ```

   Using this approach you can continue to pluralize titles for other sections.
